soccer/Processor.cpp

```diff
@@ -268,14 +266,16 @@ void Processor::run() {
         // Processor Initialization Completed
         _initialized = true;
 
-        // Log this entire frame
-        _logger->run();
+        {
+            loopMutex()->lock();
```

Comment:
This just keeps the UI from locking up quite as aggressively. It should probably be reverted.